src/main/kotlin/org/jetbrains/bio/io/Bed.kt

```diff
@@ -306,6 +306,8 @@ data class BedFormat(
  * Reads [BedEntry]-s from the supplied [reader].
  *
  * Set [stringency] to your preferred mode before starting to parse. See [Stringency] for mode explanation.
+ *
+ * Use [linesFailedToParse] property to find out how many lines were skipped due to parsing errors.
```

Comment:
I think is better to use standard KDoc notation here instead of free-format text. E.g. use `@property` for properties.

Reply:
See https://kotlinlang.org/docs/reference/kotlin-doc.html

Reply:
I can't reply to the previous comment since it's outdated, so I'll do it here:
I'd say that it's completely expected for an `Iterator` to throw a `NoSuchElementException` when queried past its contents. `java.util.Iterator#next` is actually already annotated with corresponding `@throws` clause. Kotlin implementations do it without annotations, see e.g. `ArrayDoubleIterator`.

Reply:
@dievsky 
>  it's completely expected for an Iterator to throw a `NoSuchElementException`

I meant to mention this in the documentation, not using `@throws` method signature. Ok, I could agree that we could do not mention `NoSuchElementException` because it seems to be some Kotlin implicit convention (according to `ArrayDoubleIterator` imp)

P.S. But still, KDoc is better than self-invented documentation style not supported by IDE, etc.

Reply:
@iromeo That's not just Kotlin convention, it's Java convention. And I agree with your KDoc comment, I just don't think that an expected iterator behaviour is worth mentioning there. Now if it returned `null`s, like it did before, that would be nice to specify...

Reply:
Added KDoc.